pisek/jobs/parts/task_job.py

```diff
@@ -183,3 +190,25 @@ def _diff_files(self, file_a: TaskPath, file_b: TaskPath) -> str:
             stdout=subprocess.PIPE,
         )
         return diff.stdout.decode("utf-8")
+
+    def _file_content(
+        self,
+        file: TaskPath,
+        force: bool = False,
+        tab_times: int = 1,
+        color: str = "yellow",
+        **kwargs,
+    ) -> str:
+        if force or self._env.file_contents:
+            with self._open_file(file) as f:
+                text = self._short_text(f.read().strip(), **kwargs)
+                text = colored_env(text, color, self._env)
+                for _ in range(tab_times):
+                    text = tab(text)
+            return text + "\n"
+        else:
+            self._access_file(file)  # Caching improvements
+            return ""
+
+    def _named_file(self, file: TaskPath, **kwargs) -> str:
+        return f"{file.col(self._env)}\n{self._file_content(file, **kwargs)}"
```

Comment:
This function name seems really unclear, since every file is named. How about `_quote_file_with_name` or someting similar?